apps/mobile/src/components/Community/Tabs/CommunityViewPostsTab.tsx

```diff
@@ -57,6 +63,25 @@ export function CommunityViewPostsTab({ communityRef, queryRef }: Props) {
     queryRef
   );
 
+  const isMemberOfCommunityQuery = useLazyLoadQuery<CommunityViewPostsTabQuery>(
```

Comment:
are we able to put this on the existing `query`?

Reply:
Good one! Merge it to existing query